haralyzer/assets.py

```diff
@@ -632,3 +632,84 @@ def video_load_time(self):
     @cached_property
     def html_load_time(self):
         return self._get_asset_load('html')
+
+
+class HarEntry(object):
+    """
+        An object that represent one entry in a HAR Page
+    """
+    def __init__(self, entry):
+        self.raw_entry = entry
+
+    def get_header_value(self, name):
+        """
+        Returns the header value of the header defined in ``name``
+
+        :param name: ``str`` name of the header to get the value of
+        """
+        for x in self.headers:
+            if x["name"].lower() == name.lower():
+                return x["value"]
+
+    @cached_property
+    def headers(self):
+        return self.raw_entry["request"]["headers"] + self.raw_entry["response"]["headers"]
+
+    @cached_property
+    def request(self):
+        return Request(entry=self.raw_entry["request"])
+
+    @cached_property
+    def response(self):
+        if type(self.raw_entry) == dict:
```

Comment:
Again, would suggest `isinstance` here, like `if isinstance(self.raw_entry, dict)`

Reply:
I will clean up all `type` to `isinstance`